app/src/components/widgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart.jsx

```diff
@@ -152,40 +152,80 @@ export class IssueStatisticsChart extends Component {
     type: TEST_ITEM_PAGE,
   });
 
-  getConfig = () => {
+  getChartColors(columns) {
+    const { defectTypes } = this.props;
+
+    return columns.reduce((colors, column) => {
+      const locator = getItemNameConfig(column[0]).locator;
+      const defectTypesValues = Object.values(defectTypes);
+
+      for (let i = 0; i < defectTypesValues.length; i += 1) {
+        const defect = defectTypesValues[i].find((defectType) => defectType.locator === locator);
+
+        if (defect) {
```

Comment:
```
return defect ? {
    ...colors,
    [column[0]]: defect.color 
} : colors;
```

Reply:
No, we can't break the loop on first iteration, we need to find first match and then we can break

Reply:
Oh, you right! Sorry.